analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NewGuidShouldNotBeUsed.CSharp9.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Text;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public void Foo()
+        {
+            Guid result = new(); // Noncompliant
+
+            result = default; // Noncompliant
+//                   ^^^^^^^
+
+            StringBuilder st = new(); // Checking that the rules raises issue only for the Guid class.
+            string test = default;
```

Comment:
We can add one more test with a method:
`public Guid Get() => default; // Noncompliant`